api/src/Page/DC.php

```diff
@@ -62,6 +62,8 @@ class DC extends Page
         array('/dat/:id', 'get', '_plot'),
     );
 
+    public $eVToA = 12398.4198;
```

Comment:
```suggestion
    public const $eVToA = 12398.4198;
```

Since we're using it in a SQL query in a position that is rather sensitive to injections, it's better to explicitly mark it as a constant

Reply:
That gave an error, but I've done it as a class constant in the commit below.